controllers/util/k8sutil.go

```diff
@@ -153,13 +165,13 @@ func GenerateNameForNonResilientCephBlockPoolSC(initData *ocsv1.StorageCluster)
 	return fmt.Sprintf("%s-ceph-non-resilient-rbd", initData.Name)
 }
 
-func MapCRDAvailability(ctx context.Context, clnt client.Client, log logr.Logger, crdNames ...string) (map[string]bool, error) {
+func MapCRDAvailability(ctx context.Context, clnt client.Client, crdNames ...string) (map[string]bool, error) {
```

Comment:
I don't think this function is even necessary anymore. We are using a cached client already

Reply:
From the main we are using an uncached client